src/client/components/Containers/Collapsible.tsx

```diff
@@ -1,18 +1,17 @@
 import styled from 'styled-components';
-import React, { FunctionComponent, useRef, useState, useEffect } from 'react';
-import ResizeObserver from 'resize-observer-polyfill';
-import { FlexStartColumn, ContainerProps } from './FlexContainers';
+import React, { FunctionComponent } from 'react';
 import { hexToRGB, Props as PopupProps } from './FloatingPopup';
 import DownArrow from '../../assets/img/down_arrow.svg';
 import UpArrow from '../../assets/img/up_arrow.svg';
 
+/*
 export const useMeasure = (): any => {
 	const ref = useRef<any>();
 	const [bounds, set] = useState({ height: 0, left: 0, top: 0, width: 0 });
```

Comment:
Delete it or uncomment it.

Reply:
I don't like deleting large chunks of code because we'll basically end up rewriting it. Git history is not a suitable substitute for commented-out code because noone's gonna look at git history to see if a method has been written before.